kubernetes/config-maps/concent-api/local_settings.py.j2

```diff
@@ -32,3 +32,9 @@ EMAIL_HOST_PASSWORD = EMAIL_PASSWORD
 EMAIL_USE_TLS       = {{ email.use_tls }}
 DEFAULT_FROM_EMAIL  = "{{ email.default_from_email }}"
 SERVER_EMAIL        = "{{ email.server_email }}"
+
+CONCENT_FEATURES = [
+{% for feature in features %}
+    "{{ feature }}",
+{% endfor %}
+]
```

Comment:
It's not supposed to work like this. In `concent` repo the features are a setting because it can be used in various configurations. But here we have a specific configuration. This is the config map for `concent-api` and this pod always provides two features: `"concent-api"` and `"admin-panel"`.